apps/codeimage/src/components/PropertyEditor/FrameStyleForm.tsx

```diff
@@ -27,16 +39,24 @@ export const FrameStyleForm: ParentComponent = () => {
           <SuspenseEditorItem
             fallback={<SkeletonLine width={'100%'} height={'26px'} />}
           >
-            <SegmentedField
-              adapt
+            <Select
```

Comment:
we should add the `disallowEmptySelection ` input in order to force the user to select a value

Reply:
Done